src/engine/sys/sdl_input.cpp

```diff
@@ -1032,9 +1032,8 @@ static void IN_ProcessEvents( bool dropInput )
 					Key kKeycode = IN_TranslateSDLToQ3Key( &e.key.keysym, true );
 					bool consoleFound = false;
 					for (Key k: {kScan, kKeycode} ) {
-						if ( IN_IsConsoleKey( k ) && !keys[ Key(K_ALT) ].down) {
+						if ( IN_IsConsoleKey( k ) ) {
```

Comment:
This is useful, it allows you to type in backticks and tilde into the console for any reason.

Reply:
Does it really? I don't know any OS where you can enter text while Alt is held.

Reply:
I can with Linux

Reply:
I confirm I can type `²` in console on linux. As for backticks, french layout requires the use of the `alt gr` key anyway, so I can't really tell.

Reply:
It works for me too. I thought I had tested this on Linux and it didn't work. So I put it back.